vllm/engine/arg_utils.py

```diff
@@ -567,6 +568,15 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             help='Min size of window for ngram prompt lookup in speculative '
             'decoding.')
 
+        parser.add_argument(
```

Comment:
I feel we can remove this flag completely in spec decode if your change fixes the bug.

Reply:
Done